src/Storages/PostgreSQL/PostgreSQLReplicationHandler.cpp

```diff
@@ -157,7 +157,7 @@ PostgreSQLReplicationHandler::PostgreSQLReplicationHandler(
 
     checkReplicationSlot(replication_slot);
 
-    LOG_INFO(log, "Using replication slot {} and publication {}", replication_slot, publication_name);
+    LOG_INFO(log, "Using replication slot {} and publication {}", replication_slot, doubleQuoteString(publication_name));
```

Comment:
Why do we need double quotes? Why not to use single quotes or backticks?

Reply:
For logging it does not matter what we use, but for query to postgres we should use double quotes.

Reply:
Yes, It does not matter when logging.  Using single quotes or backticks may be the same.
I saw other logs using doubleQuoteString so I used it.    :-)